package.json

```diff
@@ -83,40 +83,40 @@
     "@storybook/source-loader": "7.0.20",
     "@testing-library/dom": "9.3.0",
     "@testing-library/jest-dom": "^5.17.0",
-    "@testing-library/react": "^14.0.0",
-    "@testing-library/testcafe": "^5.0.0",
+    "@testing-library/react": "^14.1.2",
+    "@testing-library/testcafe": "^5.0.1",
```

Comment:
I'm still having issues running testcafe locally. Maybe it is this dependency?

Reply:
Nevermind, looks good. I just had to remove node_modules and do yarn cache clean